KoAR.Core/Stash.cs

```diff
@@ -4,30 +4,31 @@ namespace KoAR.Core
 {
     public class Stash
     {
+        public GameSave GameSave { get; }
```

Comment:
probably best as private readonly field. No one should be accessing this property on the item.